src/vistir/misc.py

```diff
@@ -289,11 +289,11 @@ def run(
     env = _env
```

Comment:
I think this line can be safely removed?

Reply:
At first I thought so too, but it's actually repurposed and referenced below.
```python
_env = {fs_encode(k): fs_encode(v) for k, v in _env.items()}
        for key, val in env.items():
            _env[fs_encode(key)] = fs_encode(val)
```

Reply:
I think this bug only effects python3 right?
With your PR, I think following can also be removed since the envs (os.environment and env argument) are already encoded by ```_env = {fs_encode(k): fs_encode(v) for k, v in _env.items()}```. There is no need to re-encode the env argument again.

```
for key, val in env.items():
    _env[fs_encode(key)] = fs_encode(val)
```

Reply:
Yes, now that os.environ is gone, that line can probably be removed.